include/common_components.php

```diff
@@ -5,13 +5,15 @@ function tokenSetup() {
     if (!isset($_COOKIE['spotify_token'])) { 
         if (isset($_REQUEST['code']) && isset($_REQUEST['state'])) {
             $token = requestAccessToken($_REQUEST['code'], $_REQUEST['state']); 
-            setUserSpotifyId($token, 1);  // userId is hard coded for now
-            setTotalSongs($token, 1); // userId is hardcoded, songs are only set like once an hour or once every time you log in
+            $userId = createUser($token);
+            $_SESSION['userId'] = $userId ?? createUser($token);
```

Comment:
Maybe I'm misreading this, but wouldn't the null coalescing here be redundant? We're already calling it on L8, so I think it'll get set no matter what?